src/cascadia/TerminalCore/lib/terminalcore-lib.vcxproj

```diff
@@ -49,6 +54,9 @@
     </ClCompile>
   </ItemDefinitionGroup>
 
+  <PropertyGroup Condition="'$(Configuration)'=='AuditMode'">
+    <CAExcludePath>$(SolutionDir)src\cascadia\TerminalSettings;$(CAExcludePath)</CAExcludePath>

```

Comment:
Is there a reason these two blocks aren't in one statement?

Reply:
I think it was just an artifact of figuring out how to make it work. I'll consolidate.